picard/util/checkupdate.py

```diff
@@ -102,8 +102,7 @@ def available_versions(self):
 
     def _query_available_updates(self, callback=None):
         '''Gets list of releases from specified website api.'''
-        output_text = _("Getting release information from %s." % (VERSIONS_API['host'],))
-        log.debug(output_text)
+        log.debug(_("Getting release information from {host_url}.".format(host_url=VERSIONS_API['host'],)))
```

Comment:
We don't usually translate debug messages (that would be extra work for translators).
And there's an error here:

```python
log.debug(_("Getting release information from {host_url}.".format(host_url=VERSIONS_API['host'],)))
```

Should be:

```python
log.debug(_("Getting release information from {host_url}.").format(host_url=VERSIONS_API['host']))
```

`_()` returns the translated string for `format()`, and no need of trailing comma (at least not on a single line)